buildSrc/src/test/kotlin/com/datadog/gradle/plugin/jsonschema/ModelValidationTest.kt

```diff
@@ -6,7 +6,12 @@
 
 package com.datadog.gradle.plugin.jsonschema
 
+import com.datadog.android.core.internal.utils.toJsonElement
```

Comment:
seems like this file actually not changed: I don't see these new `import`s used and indent change should be rolled back.

Reply:
hmmm don't know what happened, will have a look ;)